packages/actions/docs/07-prebuilt-actions/09-export.md

```diff
@@ -96,7 +96,7 @@ public function getColumns(): array
     return [
         ExportColumn::make('name'),
         ExportColumn::make('sku')
-            ->label('SKU')),
+            ->label('SKU'),
         ExportColumn::make('price'),
```

Comment:
how to make number format for price

Reply:
Not really the place for asking questions, but I’ve took some time to help you out. The official Filament documentation has currency formatting documented: https://filamentphp.com/docs/3.x/tables/columns/text#currency-formatting